sass/utilities/derived-variables.sass

```diff
@@ -18,7 +18,7 @@ $blue-invert: findColorInvert($blue) !default
 $purple-invert: findColorInvert($purple) !default
 $red-invert: findColorInvert($red) !default
 
-$primary-invert: $turquoise-invert !default
+$primary-invert: findColorInvert($primary) !default
```

Comment:
If this gets to be merged, it would be more consistent to do the same for all the variables below.

Reply:
I will modify afterwards, on my mobile now! You are right...